src/main/java/seedu/address/model/timetable/Module.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.timetable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents a module in NUS.
+ */
+public class Module {
+    private List<Lesson> enrolledLessons;
+
+    private final String moduleCode;
```

Comment:
What does `enrolledLessons` mean? Is it all the possible lessons within the module? Or when someone enrols in the lesson?

Reply:
@mynameizzhafeez for our context, each student who is enrolled in a Module will not be expected to be enrolled in all classes. Perhaps changing `Module` to `EnrolledModule` makes it less confusing?

Reply:
So does this class deal with the persons? Or is it a standalone thing?

Reply:
If it is standalone, then `Module` is the best name to give as it is the most straightfoward.

Reply:
I think `EnrolledModule` is better handled in `ModuleTag`.

Reply:
@mynameizzhafeez For example, we both take CS2030S, it could be interpreted as the _same_ Module in **layman**. However for our context: `Module` must be seen as such:  it should be noted that each of us have different Lab, Tutorials and Recitation slots, so think of what slots we got during MODREG, package them into `Module`. so the CS2030S `Module` that Person A has would be different from that of CS2030S `Module` that Person B has because the enrolled slots (from MODREG I guess) are different!

Reply:
@mynameizzhafeez does this clarify things?